python/neuroglancer/graph_server.py

```diff
@@ -259,10 +250,14 @@ def threshold_graph(G):
 
 def start_server(path=None):
     app = make_app(path)
-    sockets = netutil.bind_sockets(0, '')
-    server = httpserver.HTTPServer(app)
-    server.add_sockets(sockets)
-    
+    http_server = tornado.httpserver.HTTPServer(app, ssl_options={
```

Comment:
I don't think we are going for a super secure setup here, just want HTTPS for HTTP2 or some such, so default settings should be fine.